tasks/eth/fp-recovery/002-fallback-permissioned-game/input-template.json

```
@@ -0,0 +1,38 @@
+{
+  "chainId": 1,
+  "metadata": {
+    "name": "Deputy Guardian - Fall back to `PermissionedDisputeGame`",
+    "description": "This batch udates the `respectedGameType` to `PERMISSIONED_CANNON` in the `OptimismPortal`. This action requires all in-progress withdrawals to be re-proven against a new `PermissionedDisputeGame` that was created after this update occurs."
```

nit:
```suggestion
    "description": "This batch updates the `respectedGameType` to `PERMISSIONED_CANNON` in the `OptimismPortal`. This action requires all in-progress withdrawals to be re-proven against a new `PermissionedDisputeGame` that was created after this update occurs."
```

Probably a typo in the original sepolia version too.